test/bigtest/network/factories/addresse.js

```diff
@@ -0,0 +1,18 @@
+import { Factory, faker } from '@bigtest/mirage';
```

Comment:
This file should be named `address.js`. Likewise for `.../network/models/addresse.js`. Everything else looks great!

Reply:
Done!